doc/source/api/visualization/index.rst

```diff
@@ -6,106 +6,65 @@ Visualization
 postprocessing capabilities, PyVista, or Matplotlib integration.
 
-Fluent TUI command example
---------------------------
+Fluent command example
+----------------------
 
 Here visualization objects are constructed within Fluent. You can use
 standard Fluent commands to write graphics to a file.
 
 .. code:: python
 
-  solver_session.tui.display.objects.contour['contour-1'] = {'boundary_values': True, 'color_map': {'color': 'field-velocity', 'font_automatic': True, 'font_name': 'Helvetica', 'font_size': 0.032, 'format': '%0.2e', 'length': 0.54, 'log_scale': False, 'position': 1, 'show_all': True, 'size': 100, 'user_skip': 9, 'visible': True, 'width': 6.0}, 'coloring': {'smooth': False}, 'contour_lines': False, 'display_state_name': 'None', 'draw_mesh': False, 'field': 'pressure', 'filled': True, 'mesh_object': '', 'node_values': True, 'range_option': {'auto_range_on': {'global_range': True}}, 'surfaces': [2, 5]}
-  solver_session.tui.display.objects.contour['contour-1']()
-  solver_session.tui.display.objects.contour['contour-1'].field.set_state('velocity-magnitude')
-  solver_session.tui.display.objects.contour['contour-1'].field()
-  solver_session.tui.display.objects.contour['contour-1'].color_map.size.set_state(80.0)
-  solver_session.tui.display.objects.contour['contour-1'].color_map.size()
-  solver_session.tui.display.objects.contour['contour-1'].rename('my-contour')
-  del solver_session.tui.display.objects.contour['my-contour']
+  solver_session.settings.results.graphics.contour['contour-1'] = {'boundary_values': True, 'color_map': {'color': 'field-velocity', 'font_automatic': True, 'font_name': 'Helvetica', 'font_size': 0.032, 'format': '%0.2e', 'length': 0.54, 'log_scale': False, 'position': 1, 'show_all': True, 'size': 100, 'user_skip': 9, 'visible': True, 'width': 6.0}, 'coloring': {'smooth': False}, 'contour_lines': False, 'display_state_name': 'None', 'draw_mesh': False, 'field': 'pressure', 'filled': True, 'mesh_object': '', 'node_values': True, 'range_option': {'auto_range_on': {'global_range': True}}, 'surfaces': [2, 5]}
+  solver_session.settings.results.graphics.contour['contour-1']()
+  solver_session.settings.results.graphics.contour['contour-1'].field = 'velocity-magnitude'
+  solver_session.settings.results.graphics.contour['contour-1'].field()
+  solver_session.settings.results.graphics.contour['contour-1'].color_map.size.set_state(80.0)
+  solver_session.settings.results.graphics.contour['contour-1'].color_map.size()
+  solver_session.settings.results.graphics.contour['contour-1'].rename('my-contour')
+  del solver_session.settings.results.graphics.contour['contour-1']['my-contour']
 
-PyVista example (graphics)
---------------------------
+Visualization example
+---------------------
 
 Here field data is extracted from the Fluent session into the Python
-environment. PyVista is then used to visualize the extracted data.
+environment. Visualization is then used to visualize the extracted data.
```

Comment:
```suggestion
environment. Visualization is then used to present the extracted data.
```

Reply:
@prmukherj Many sentences can be improved which is best done during the self-review stage using the tools discussed.